[2025-03-17T22:33:50.275+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_map.transform_phase_1_task manual__2025-03-17T22:29:46.221740+00:00 [queued]>
[2025-03-17T22:33:50.334+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_map.transform_phase_1_task manual__2025-03-17T22:29:46.221740+00:00 [queued]>
[2025-03-17T22:33:50.335+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-03-17T22:33:50.526+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): transform_phase_1_task> on 2025-03-17 22:29:46.221740+00:00
[2025-03-17T22:33:50.533+0000] {standard_task_runner.py:60} INFO - Started process 19159 to run task
[2025-03-17T22:33:50.561+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'Google_map', 'transform_phase_1_task', 'manual__2025-03-17T22:29:46.221740+00:00', '--job-id', '223', '--raw', '--subdir', 'DAGS_FOLDER/google_map_dag_etl.py', '--cfg-path', '/tmp/tmp9k000xwz']
[2025-03-17T22:33:50.574+0000] {standard_task_runner.py:88} INFO - Job 223: Subtask transform_phase_1_task
[2025-03-17T22:33:51.379+0000] {task_command.py:423} INFO - Running <TaskInstance: Google_map.transform_phase_1_task manual__2025-03-17T22:29:46.221740+00:00 [running]> on host DESKTOP-AVAHALM.
[2025-03-17T22:33:52.824+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='master_m2si' AIRFLOW_CTX_DAG_ID='Google_map' AIRFLOW_CTX_TASK_ID='transform_phase_1_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-17T22:29:46.221740+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-17T22:29:46.221740+00:00'
[2025-03-17T22:33:52.827+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-17T22:33:52.829+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd ~/data_warehouse_project && dbt run --profiles-dir ~/.dbt --models cleaned_reviews']
[2025-03-17T22:33:52.976+0000] {subprocess.py:86} INFO - Output:
[2025-03-17T22:34:22.281+0000] {subprocess.py:93} INFO - [0m22:34:22  Running with dbt=1.9.0-b2
[2025-03-17T22:34:27.204+0000] {subprocess.py:93} INFO - [0m22:34:27  Registered adapter: postgres=1.8.2
[2025-03-17T22:34:30.185+0000] {subprocess.py:93} INFO - [0m22:34:30  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-03-17T22:34:30.186+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-03-17T22:34:30.186+0000] {subprocess.py:93} INFO - - models.data_warehouse_project.example
[2025-03-17T22:34:32.068+0000] {subprocess.py:93} INFO - [0m22:34:32  Found 6 models, 1 source, 424 macros
[2025-03-17T22:34:32.132+0000] {subprocess.py:93} INFO - [0m22:34:32
[2025-03-17T22:34:32.134+0000] {subprocess.py:93} INFO - [0m22:34:32  Concurrency: 4 threads (target='dev')
[2025-03-17T22:34:32.136+0000] {subprocess.py:93} INFO - [0m22:34:32
[2025-03-17T22:34:34.868+0000] {subprocess.py:93} INFO - [0m22:34:34  1 of 1 START sql table model public.cleaned_reviews ............................ [RUN]
[2025-03-17T22:34:37.397+0000] {subprocess.py:93} INFO - [0m22:34:37  1 of 1 OK created sql table model public.cleaned_reviews ....................... [[32mSELECT 7[0m in 2.43s]
[2025-03-17T22:34:37.568+0000] {subprocess.py:93} INFO - [0m22:34:37
[2025-03-17T22:34:37.584+0000] {subprocess.py:93} INFO - [0m22:34:37  Finished running 1 table model in 0 hours 0 minutes and 5.43 seconds (5.43s).
[2025-03-17T22:34:38.197+0000] {subprocess.py:93} INFO - [0m22:34:38
[2025-03-17T22:34:38.201+0000] {subprocess.py:93} INFO - [0m22:34:38  [32mCompleted successfully[0m
[2025-03-17T22:34:38.203+0000] {subprocess.py:93} INFO - [0m22:34:38
[2025-03-17T22:34:38.206+0000] {subprocess.py:93} INFO - [0m22:34:38  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-03-17T22:34:40.001+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-17T22:34:40.243+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Google_map, task_id=transform_phase_1_task, execution_date=20250317T222946, start_date=20250317T223350, end_date=20250317T223440
[2025-03-17T22:34:40.332+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-17T22:34:40.891+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
